website/docs/installation/windows-install/index.md

```diff
@@ -1,28 +1,229 @@
+
+Main benefits are:
+
+- Ease of use.
+- [Windows Subsystem for Linux version 2 (WSL 2)](https://learn.microsoft.com/en-us/windows/wsl/about#what-is-wsl-2) native virtualization performance.
+
+Check that your environment has:
+
+- 6 GB RAM for the Podman Machine.
+- [WSL prerequisites](https://learn.microsoft.com/en-us/windows/wsl/troubleshooting#error-0x80370102-the-virtual-machine-could-not-be-started-because-a-required-feature-is-not-installed):
+  - The Windows user has administrator privileges.
+  - Windows 64bit.
+  - Windows 10 Build 19043 or greater, or Windows 11.
+  - On a virtual machine: [Nested Virtualization enabled](https://learn.microsoft.com/en-us/virtualization/hyper-v-on-windows/user-guide/nested-virtualization#configure-nested-virtualization).
+
+<details>
+<summary>
+Alternatively, consider installing the Podman Preset in OpenShift Local when you require one of following:
+
+- Full control on the virtual machine.
+- HyperV virtual machine, rather than WSL 2.
+- The Windows user is a member of the _Hyper-V Administrators_ group, rather than having administrator privileges.
+  The user can add a virtual machine, start or stop this virtual machine, and access the resources related to this virtual machine.
+- Windows 10 version 1709 or later, Enterprise or Professional edition, or Windows 11, Enterprise or Professional edition.
+
+</summary>
+
+1. Install the OpenShift Local extension:
+
+   Go to the **Dashboard**.
+
+   In the **Featured extensions > OpenShift Local** tile, click the **<Icon icon="fa-solid fa-download" size="lg" />** button.
+
+1. Install OpenShift Local:
+
+   In the **OpenShift Local** tile, click the **<Icon icon="fa-solid fa-rocket" size="lg" /> Install** button.
+
+   Validate all confirmation screens.
+
+1. Selecting a preset is a mandatory step.
+   However the **Podman** preset is not visible in Podman Desktop UI.
+
+   Select **Microshift**.
+
+   OpenShift Local downloads the preset in the background. However you do not intent to use this preset.
+
+1. Restart your computer.
+
+1. Use the OpenShift Local CLI to select the **Podman** preset.
+
+   To run the next commands, open the Command Prompt.
+
+1. Select the [Podman container runtime preset](https://access.redhat.com/documentation/en-us/red_hat_openshift_local/2.12/html/getting_started_guide/using_gsg#about-presets_gsg):
+
+   ```shell-session
+   > crc config set preset podman
+   ```
+
+1. Set up your host machine for Red Hat OpenShift Local, and download the Podman bundle:
+
+   ```shell-session
+   > crc setup
+   ```
+
+1. Start the Red Hat OpenShift instance:
+
+   ```shell-session
+   > crc start
+   ```
+
+1. The Podman Desktop **Dashboard** displays: _OpenShift Local is running Version x.y.z (Podman)_.
+
+1. Configure your terminal to use the Podman CLI:
+
+   ```shell-session
+   > eval $(crc podman-env)
+   ```
+
+</details>
+
```

Comment:
@gbraad @slemeur Should we entirely remove this, because Podman Desktop does not display the resources properly, and the feature will be removed in the future?

```suggestion
```

Alternatively, we need to clarify what a user can expect.

Reply:
@gbraad and @slemeur agreed.